src/main/java/com/google/devtools/build/lib/sandbox/cgroups/controller/v2/UnifiedMemory.java

```diff
@@ -40,6 +40,8 @@ public Memory child(String name) throws IOException {
   @Override
   public void setMaxBytes(long bytes) throws IOException {
     Files.writeString(path.resolve("memory.max"), Long.toString(bytes));
+    // Also set swap max to 0 so that memory.max is respected.
```

Comment:
I'd rephrase this to describe the problem we're facing here and what we'd like to achieve.

For instance, "Set swap to 0 so that it doesn't unexpecedly consume more than the memory limit when swap is enabled." Feel free to rephrase and shorten the description.

And please also apply the same comment to the other file.